producer.go

```diff
@@ -283,6 +318,8 @@ type jobControlAction string
 
 const (
 	jobControlActionCancel jobControlAction = "cancel"
+	jobControlActionPause  jobControlAction = "pause"
+	jobControlActionResume jobControlAction = "resume"
```

Comment:
If all these keep sharing, we definitely want a rename of these at least, e.g.:

`Cancel` -> `JobCancel`
`Pause` -> `QueuePause`
`Resume` -> `QueueResume`

Reply:
Just a bump on this one. The topic rename looks great, but given we're still sharing, I think it makes sense to be a bit more explicit about each action by giving it a noun. Also, we can probably strip "job" from the beginning of everything now that the topic's been renamed.

What do you think of this?

``` go
const (
	controlActionCancel controlActionJobCancel   = "job_cancel"
	controlActionPause  controlActionQueuePause  = "queue_pause"
	controlActionResume controlActionQueueResume = "queue_resume"
)
```